feat: validate GitHub integration token

Roming22 opened the pull request:
Tokens can expire, so it's useful to check their validity.

rh-pre-commit.version: 2.3.2
rh-pre-commit.check-secrets: ENABLED

lcarva commented on installer/charts/rhtap-integrations/templates/tests/test.yaml:
```diff
@@ -0,0 +1,64 @@
+    {{- $scmSecretObj := (lookup "v1" "Secret" .Release.Namespace $scmSecretName) }}
+    {{- if $scmSecretObj }}
+    - name: test-{{ $scm }}-token
+      image: quay.io/codeready-toolchain/oc-client-base:latest
```
Can we use an image from either registry.redhat.io or registry.access.redhat.com?

This will run in the user's infrastructure.

Roming22 replied:
Done

lcarva commented on installer/charts/rhtap-integrations/templates/tests/test.yaml:
```diff
@@ -0,0 +1,64 @@
+    # Make sure there's at least one container
+    #
+    - name: no-op
+      image: quay.io/codeready-toolchain/oc-client-base:latest
```
One more :smile_cat:

Roming22 replied:
done

Roming22 committed:
feat: validate GitHub integration token

Tokens can expire, so it's useful to check their validity.

rh-pre-commit.version: 2.3.2
rh-pre-commit.check-secrets: ENABLED

Roming22 commented:
/retest

xinredhat commented:
/retest

xinredhat committed:
Merge branch 'main' into feat/github/validate-token

xinredhat commented:
/retest

xinredhat commented:
/retest
